inspirehep/records/indexer/base.py

```diff
@@ -117,10 +117,10 @@ def bulk_action(self, record_uuid):
                     # When record is not in es then dsl is throwing TransportError(404)
                     record._index(force_delete=True)
                 except TransportError:
-                    logger.info(f"Record not in es {record.id}")
+                    LOGGER.warning("Record not in es %r", record.id)
```

Comment:
we could unify the messages for the same use cases, maybe like this https://github.com/inspirehep/inspirehep/pull/350/files#diff-cf7cdd189453e9423ad6832b68054da5R93

Reply:
:+1: